org.electronjs.Electron2.BaseApp.yml

```
@@ -85,3 +84,25 @@ modules:
         x-checker-data:
           type: git
           tag-pattern: ^v([\d.]+)$
+
+  - name: setup-separate-locales
+    buildsystem: simple
+    build-commands:
+      - install -m755 setup-separate-locales /app/bin/setup-separate-locales
+    sources:
+      - type: script
+        dest-filename: setup-separate-locales
+        commands:
+          - |
+            APP_LOCALES_DIR=$1/locales
+            RUNTIME_LOCALES_DIR=${FLATPAK_DEST}/share/runtime/locale
+            if [ -d ${APP_LOCALES_DIR} ]
+            then
+                for lang in ${APP_LOCALES_DIR}/*.pak
+                do
+                locale="$(basename -s .pak $lang)"
+                install -Dm644 -t "${RUNTIME_LOCALES_DIR}/${locale%%-*}/" "$lang"
+                ln -sf "${RUNTIME_LOCALES_DIR}/${locale%%-*}/$(basename $lang)" "${APP_LOCALES_DIR}/$(basename $lang)"
+                done
+            fi
```

Since electron apps dirs are often named after the app, it's not unlikely for the dir to have spaces in its name, so we should quote all the paths.
```suggestion
            APP_LOCALES_DIR="$1/locales"
            RUNTIME_LOCALES_DIR="${FLATPAK_DEST}/share/runtime/locale"
            if [ -d "${APP_LOCALES_DIR}" ]
            then
                for lang in "${APP_LOCALES_DIR}"/*.pak
                do
                locale="$(basename -s .pak "$lang")"
                install -Dm644 -t "${RUNTIME_LOCALES_DIR}/${locale%%-*}/" "$lang"
                ln -sf "${RUNTIME_LOCALES_DIR}/${locale%%-*}/$(basename "$lang")" "${APP_LOCALES_DIR}/$(basename "$lang")"
                done
            fi
```